app/apps/server/bridges/scheduler.js

```diff
@@ -56,30 +105,62 @@ export class AppSchedulerBridge {
 		}
 	}
 
-	async scheduleRecurring(job, appId) {
-		this.orch.debugLog(`The App ${ appId } is scheduling a recurring job`, job);
-		await this.startScheduler();
-		await this.scheduler.every(job.interval, job.id, job.data || {});
+	/**
+	 * Schedules a registered processor to run recurrently according to a given interval
+	 *
+	 * @param {Object} job
+	 * @param {string} job.id The processor's id
+	 * @param {string} job.interval When the processor will be re executed
+	 * @param {Object} [job.data] An optional object that is passed to the processor
+	 * @param {string} appId
+	 *
+	 * @returns Promise<void>
+	 */
+	async scheduleRecurring({ id, interval, data }, appId) {
+		this.orch.debugLog(`The App ${ appId } is scheduling a recurring job`, id);
+		try {
+			await this.startScheduler();
+			const job = this.scheduler.create(id, data || {});
+			job.repeatEvery(interval, { skipImmediate: true });
```

Comment:
Hard coding the option limits the apps, it would be better if we added a `skipImmediate` when scheduling in the app side